sdk/tenderly-utils/src/Tenderly.ts

```diff
@@ -47,7 +47,10 @@ export class Tenderly {
    * @name createFork
    * @description Creates a new Tenderly fork
    */
-  async createFork(params: { chainInfo: IChainInfo; atBlock: number }): Promise<TenderlyFork> {
+  async createFork(params: {
+    chainInfo: IChainInfo
+    atBlock: number | 'latest'
```

Comment:
Does it make sense to set 'latest' as default value?

Reply:
I wouldn't mind, but I'm not sure how to do it due to `params` being an object. I cannot assign individual values to the object attributes, as far as I know. Let me know if you know a way

Reply:
What I've ended up doing:

```
  async createFork(params: {
    chainInfo: IChainInfo
    atBlock?: number | 'latest'
  }): Promise<TenderlyFork> {
    return TenderlyFork.create({
      tenderlyAccessKey: this.tenderlyAccessKey,
      tenderlyApiUrl: this.tenderlyApiUrl,
      chainInfo: params.chainInfo,
      atBlock: params.atBlock || 'latest',
    })
  }
```

I think it is the best we can do